File: Modul2/Python/PRAK205-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
import math

A, B = map(float, input().split())

C = math.sqrt(B**2 - A**2)
keliling = A + B + C
luas = 1/2 * A * C

print(f"\nAlas = {C:.0f} cm")
print(f"Tinggi = {A:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul2/C/PRAK205-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>
#include <math.h>

int main(){
    double A;
    double B;

    scanf("%lf %lf", &A, &B); printf("\n");

    double C = sqrt(pow(B, 2) - pow(A, 2));
    double keliling = A + B + C;
    double luas = (double) 1/2*A*C;

    printf("Alas = %g cm\n", C);
    printf("Tinggi = %g cm\n", A);
    printf("Keliling = %g cm\n", keliling);
    printf("Luas = %g cm^2", luas);

    return 0;
}